modules/core/index.js

```diff
@@ -0,0 +1,10 @@
+export { Connection } from './connection';
+export { Difference } from './difference';
+export { Entity } from './entity';
+export { Graph } from './graph';
+export { History } from './history';
+export { iD_Node as Node } from './node';
```

Comment:
Not it didn't break anything. But it might break if we want the use `Node API` inside the scope of `./node`. But that is rarely going to happen. So, I guess we should stick to `Node` then...